ListableUI/Sources/ListView/ListView.swift

```diff
@@ -1333,6 +1333,61 @@ extension ListView : ReorderingActionsDelegate
         }
     }
     
+    func accessibilityMove(item: AnyPresentationItemState, direction: ReorderingActions.AccessibilityMoveDirection) -> Bool {
+        guard let indexPath = self.storage.presentationState.indexPath(for: item),
+        self.dataSource.collectionView(self.collectionView, canMoveItemAt: indexPath) else {
+            return false
+        }
+
+        let destinationPath : IndexPath
+        switch direction {
+        case .up:
+            // Moving an item up means decrementing the index.
+            if indexPath.row == 0 {
+                // First item in section, we should go to the previous section
+                if indexPath.section > 0 {
+                    let newSection = indexPath.section - 1
+                    let rowInNewSection = self.storage.allContent.sections[indexPath.section - 1].count
+                    destinationPath = IndexPath(row: rowInNewSection, section:newSection )
+                }
+                else {
+                    // Unable to move up, we are item 0,0.
+                    return false
+                }
+            } else {
+                destinationPath = IndexPath(row: indexPath.row - 1, section: indexPath.section)
+            }
+
+        case .down:
+            // Moving an item down means incrementing the index.
+            if indexPath.row == storage.allContent.sections[indexPath.section].count - 1 {
+                // we are the last item our section, lets see if there's another section we can move down to
+                if storage.allContent.sections.count > indexPath.section {
```

Comment:
Maybe I'm wrong, but don't we need to decrement the sections.count by 1 for this comparison, since `indexPath.section` is zero indexed? e.g., if there's three sections, and we're on the last section, the section will be 2, the count will be 3, so `count > section = true` but we can't increase the section past 2

```suggestion
                if storage.allContent.sections.count - 1 > indexPath.section {
```

Reply:
Yep, of course.